### PR TITLE
Initial roi-feature in model app

### PR DESCRIPTION
**Pull request by leoarc**
## Description
The roi-feature is added in the model app.

## How Has This Been Tested?
OS : Ubuntu 20.04
Browser : Firefox 77.0.1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/model/model.js [hunk 1]
```diff
@@ -994,3 +1072,281 @@ function downloadCSV(filename) {
...
+          data[key].weightSpecsBytes) / (1024*1024);
+        const row = table.insertRow();
+        let classes; let inputShape; let td;
+
+        if (name.slice(0, 4) == 'pred') {
+          store.get(name).onsuccess = function(e) {
+            classes = (e.target.result.classes.join(', '));
+            inputShape = e.target.result.input_shape.slice(1, 3).join('x');
+            td = row.insertCell();
+            td.innerHTML = name.split('/').pop().split('_').splice(2).join('_').slice(0, -3);
```
Comment (birm) [inline apps/model/model.js:1101]: would innertext be safer?

Comment (leoarc) [reply apps/model/model.js:1101]: Yes that would be much better. I'll change it.

**Comment by birm** [comment]
With a *super* quick pass, looks ok to me. Thanks for the innertext change!
I can't seem to add you as a reviewer, @Insiyaa but I can bug you in this comment. We need to resolve those permissions for you.

**Comment by Insiyaa** [comment]
Yes, I will go through the addition

**Comment by leoarc** [comment]
Yes that is a great idea.  I'll add that in the next PR.

**Pull request merged**